designcompose/src/main/java/com/android/designcompose/Utils.kt

```diff
@@ -1230,42 +1230,84 @@ internal fun Background.asBrush(document: DocContent, density: Float): Pair<Brus
         }
         is Background.LinearGradient -> {
             val linearGradient = this
-            return Pair(
-                RelativeLinearGradient(
-                    linearGradient.color_stops.map { convertColor(it.field1) }.toList(),
-                    linearGradient.color_stops.map { it.field0 }.toList(),
-                    start = Offset(linearGradient.start_x, linearGradient.start_y),
-                    end = Offset(linearGradient.end_x, linearGradient.end_y)
-                ),
-                1.0f
-            )
+            when (linearGradient.color_stops.size) {
+                0 -> {
+                    Log.e(TAG, "No stops found for the linear gradient")
+                    return null
```

Comment:
This should never happen because Rust will return `Background::None` right? (good to have it still, just checking)

Reply:
Yeah, I think you get here if you have an old serialized doc.

Reply:
👍